core/jobs/batch_jobs/tabs_collapsables_validation_jobs.py

```diff
@@ -147,25 +158,40 @@ def _filter_invalid_rte_values(html, states_with_errored_values):
                         'text attr not in skillreview tag'
                     )
 
+                try:
+                    if tag['skill_id-with-value'].strip() in (
+                        '&quot;&quot;', '', '\'\'', '\"\"'):
+                        states_with_errored_values.append(
+                            'skill_id attr empty in skillreview tag'
+                        )
+
+                except Exception:
+                    states_with_errored_values.append(
+                        'skill_id attr not in skillreview tag'
+                    )
+
             elif tag.name == 'oppia-noninteractive-video':
+                start_value = 0
+                end_value = 0
                 try:
-                    tag['start-with-value']
+                    start_value = tag['start-with-value']
 
                 except Exception:
                     states_with_errored_values.append(
                         'start value invalid'
                     )
 
                 try:
-                    tag['end-with-value']
+                    end_value = tag['end-with-value']
 
                 except Exception:
                     states_with_errored_values.append(
                         'end value invalid'
                     )
 
                 try:
-                    if tag['autoplay-with-value'] not in ('true', 'false'):
+                    if tag['autoplay-with-value'] not in (
+                        'true', 'false', '\"true\"', '\"false\"', True, False):
```

Comment:
I'm guessing you did this because you haven't looked into what the possible values can be? I get what you're doing and I guess it's fine, but in the future, look into it first and then write the code.

Reply:
I have looked at the values and the values can only be of the form `"true" or "false"` but Sean mentioned previously that if there is any possibility from which we can depict the original intention of the user we should do so.